licenses/_headers/license-prepend-missing.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2017, UT-BATTELLE, LLC
+# All rights reserved.
+#
+# This software is released under the BSD license detailed
+# in the file licenses/A-PRIME_LICENSE
+#
+# Copyright (c) 2017,  Los Alamos National Security, LLC (LANS)
```

Comment:
@kevans32, @milenaveneziani and @jhkennedy, does this seem like a good way to handle licenses for files that are originally A-Prime but which I have modified in the MPAS-Analysis copies?  If not, I'm open to suggestions.

Reply:
I think that's a fine way to handle it. 

It looks like the A-Prime files are just the license header scripts? Those actually originated in LIVVkit, and were imported into A-Prime (an then modified a little). I'm not sure which would be better to credit. @kevans32 what do you think?

Reply:
If you'd rather have LIVVkit get credited, you might want to include the LIVVkit license in A-Prime and reference it in these scripts, perhaps similarly to how I've done it.  Otherwise, others who copy the scripts from A-Prime will credit A-Prime as I have done...

Reply:
@jhkennedy wrote the original scripts, so in the end its his call. We decided that for A-PRIME, since he has been helpful to us with it, to add him as a co-developer of A-PRIME and have those files be part of A-PRIME (since Joe developed them they can be equally shared authorship in both repos. So in theory since Joe is also part of MPAS-Analysis that could be true here as well. But maybe that is not the best way? His co-developer status does not hinge on the license files so we could update the files with however folks decide to proceed and it works. But I think doing some thing consistent with both A-PRIME and MPAS-A is good.

Reply:
PS the license approval process took >3months for both LIVVKit and A-PRIME here. Good luck!

Reply:
....and I just looked at the OSTI release and Joe is not there (since I submitted before we did the license files and Joe was contributing to it ( think? maybe? @jhkennedy  helps a lot on a lot of things...). So, Joe we will have to add you on follow-on efforts as a contributor). Given this, I think we should update the A-PRIME and MPAS-A files to point to LIVVkit to give proper credit. With agreement I can take care of that.

Reply:
@xylar after thinking on it a bit, I think it makes the most sense to credit A-Prime here (as done) since that *is* where the scripts were from. @kevans32 we can discuss how to best credit LIVVkit in A-Prime.